src/Script/RamCalculations.ts

```diff
@@ -338,7 +344,7 @@ function parseOnlyCalculateDeps(code: string, currentModule: string): ParseDepsR
     Object.assign(
       {
         ImportDeclaration: (node: Node, st: State) => {
-          const importModuleName = node.source.value;
+          const importModuleName = resolveFilePath(node.source.value, filename) as FilePath;
```

Comment:
Instead of asserting that this is a FilePath instead of null, why not throw if it ends up being null so we can control the error message?

Reply:
oh yeah that was just to get it to work
ill it cleanup today